### DISCUSSION TITLE
Allowing focus to escape and closing the popover (floating-ui)

### DISCUSSION THREAD

**Discussion by ally-bryand** [floating-ui/floating-ui#1717] [Q&A] [answered]
Hi all, 

I need to allow users to tab out of the popover, and have the popover close when they do. I set `allowEscape` to `true` for the `useListNavigation` hook, which allows the focus to escape the modal, as I want. But this doesn't trigger the popover to close. How can I achieve this?

**Comment by atomiks** [comment, marked as answer]
Are you using `<FloatingFocusManager>`? `modal={false}` will do that.

**Reply by ally-bryand** [reply]
Thanks so much for the prompt reply. Yes, that did exactly what I needed, and without having to enable the `allowEscape` option. I did notice that if a user presses `shift+tab` on the first option, focus goes back to the triggering element, but the popover doesn't close. Is that intended?

**Reply by atomiks** [reply]
I think it is, but it's a bit ambiguous as to what should happen in that case. If they press `shift+tab` once again, to move focus away from the reference element, you might want to add a `keydown` listener and check for `event.key === 'Tab'` and set the open state to false.

**Reply by ally-bryand** [reply]
I'll try using my own event listeners to handle that scenario. Thanks so much!
